buildpacks/hello-world/bin/build

```diff
@@ -18,6 +18,15 @@ if compgen -G "${env_dir}/*" > /dev/null; then
 fi
 export | sed 's/^/       /'
 
+# VOLUMES
+volume_file=/platform/volume/volume_contents.txt
+if [[ -f "$volume_file" ]]; then
+  echo "     reading volume contents:"
+  echo "         $(cat /platform/volume/volume_contents.txt)"
+else
+  echo "     no volumes found under /platform/volume"
+fi
+
```

Comment:
Instead of looking for a very specific file how would you feel about listing all files provided by the platform? 

Reasoning: I don't think we should be tying a general use sample buildpack to a very specific example.

```suggestion
# PLATFORM DIR
echo "     listing platform provided contents:"
echo "         $(ls -al $2)"
```